speed-dreams/source-2.2.3/src/libs/robottools/rthumandriver.cpp

```diff
@@ -594,6 +606,11 @@ void HumanDriver::init_track(int index,
  */
 void HumanDriver::new_race(int index, tCarElt* car, tSituation *s)
 {
+    // SIMULATED DRIVING ASSISTANCE: construct recorder when starting a race
+    // To record uncomment the #define RECORD_SESSION 1 in backend/ConfigEnums.h
+#if RECORD_SESSION == 1
+    recorder = new Recorder();
```

Comment:
Don't forget to also delete this object after the race is done! Otherwise we have a memory leak.